includes/apple-exporter/components/class-quote.php

```diff
@@ -236,7 +236,7 @@ protected function build( $html ) {
 		preg_match( $string_match, $html, $matches );
 		$this->text_alignment = 3 === count( $matches ) && $matches[1] ? $matches[1] : 'left';
 		$this->text_alignment = 'wide' === $this->text_alignment ? 'center' : $this->text_alignment;
-		$text = isset( $matches[2] ) ? $matches[2] : $matches[1];
+		$text                 = isset( $matches[2] ) ? $matches[2] : $matches[1];
```

Comment:
Could simplify this to 
```suggestion
		$text                 = $matches[2] ?? $matches[1];
```

Reply:
I'd love to do that, but that's PHP7 wizardry, and we support down to PHP 5.6 on this plugin.